### PR TITLE
Microverse Project - NYT-Magazine Collaborative Project (Positioning and Floating Elements)

### PR DESCRIPTION
**Pull request by Rocio01**
The purpose of this project is to make a clone of the "New York Times Magazine" web page.
If this pull request is reviewed and approved, it will be merge to the master branch after which a live demo link will be generated and updated on the "ReadMe" file.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+        <a href="#"><i class="fas fa-share"></i></a>
+      </div>
+      <div class="social">
+        <a href="#"> <i class="far fa-bookmark"></i> </a>
+      </div>
+      <div class="social">
+        <a href="#"> <i class="far fa-comment-alt"></i> </a>
+      </div>
+    </div>
+    <div class="article">
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<article>``` instead of ```<div class="article">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, great job with this project, we passed validator check, linter and we have a good README file :slightly_smiling_face: However, we need to make some changes in order to improve our code. Kindly check the comments below so we can get our project approved: :+1: 
- Let us add a project-specific description to our README file and our PR(Pull Request). For example; Embedding images, Working with forms (buttons in this case), Positioning elements with float, flex and grid, The box model
- ``` Footer ```, the footer should also have the fixed feature as implemented for the nav
fp
- Pull Request should have a meaningful title and a short description. Kindly edit the PR and apply the changes

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Hands down this is one of the best projects I have ever seen 🙌👌. Great job team!

### We need to make some changes before we can move ahead.

- The subscribe and login buttons in the navbar must be exactly like the original.
### Clone

Annotation 2020-06-18 232428

### Original

Annotation 2020-06-18 232408

- There are some CSS property issues in your project. Check the comments.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+
+.button {
+  text-align: center;
+  background-color: rgb(18, 18, 18);
+  color: #fff;
+  font-size: 1em;
+  height: 32px;
+  letter-spacing: 0.01em;
+  text-transform: none;
+  border-color: 1px solid #000;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: This is an invalid property. When you are declaring border-color property, only put the color values. If you want to use other border properties, use the short-hand `border` property.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+  border-radius: 3px;
+  margin-right: 1px 0 0;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: `margin-right` only accepts one value. Use short-hand `margin` property to specify values of margins for other sides.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,772 @@
...
+
+.article-text {
+  margin: 20px 30% 20px 30%;
+  font-size: 1em;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+p {
+  display: block;
+  margin-inline-end: 0;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: This property doesn't exist.

File: README.md [hunk 5]
```diff
@@ -1,57 +1,65 @@
...
-- technologies used
+- HTML
+- CSS,
 
-## Live Demo
+## Live Demo pending approval....
 
 [Live Demo Link](https://livedemo.com)
-
+[Github Repository Link](https://github.com/Rocio01/PositioningandFloatingElements/tree/feature-branch)
```
Comment (tirthajyoti-ghosh) [inline README.md]: Also, your README file doesn't contain a live demo link. Whether your project is approved or not you must put a live demo link. Update the live demo link every time you make any changes.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Take care of these invalid CSS properties. And don't forget to put the live demo link in your README.

Happy coding 👨‍💻

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! Your project looks amazing! It is definitely very impressive 🌟🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

_P.S. Please check my comments for additional **optional** changes you may want to make before merging_ 😄 

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,775 @@
...
+  height: 350px;
+  border: none;
+  margin-top: 20px;
+}
+
+.iframecaption {
+  color: rgb(105, 102, 102);
+  font-family: georgia, "times new roman", times, serif;
+  font-size: 0.9375rem !important;
+  line-height: 1rem !important;
```
Comment (BrittanyBlake) [inline css/style.css:452]: ### [OPTIONAL]

- Try not to use `!important` in your CSS. If you need to override a style, try to using selectors with a higher specificity.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,775 @@
...
+  width: 100%;
+  margin-bottom: 15px;
+}
+
+.figcaption {
+  color: rgb(105, 102, 102);
+  font-family: georgia, "times new roman", times, serif;
+  font-size: 0.9375rem !important;
+  line-height: 1rem !important;
```
Comment (BrittanyBlake) [inline css/style.css:461]: ### [OPTIONAL]

- Try not to use `!important` in your CSS. If you need to override a style, try to using selectors with a higher specificity.

**Pull request merged**